DOC.md

```diff
@@ -3689,6 +3689,26 @@ These are the settings you can provide to `luasnip.setup()`:
       	return snip.insert_nodes[0]
       end
       ```
+    - `indent`: bool?, defaults to `true`. Whether LuaSnip will try to add
+      additional indents to fit current indent level in snippet expanding. This
+      option is useful when some LSP server already take indents into
+      consideration. In such cases, LuaSnip should not try to add additional
+      indents. If you are using `nvim-cmp`, sample config:
+      ```lua
```

Comment:
Very nice!!
Only our doc-converter seems to mess up here, could you try if adding a line between "..config:" and "```lua" fixes this?

Reply:
Seems good now!

Reply:
Yup :ok_hand: